README.md

```diff
@@ -133,8 +131,10 @@ S3 bucket, and if private credentials need adding or updating, they need to be u
 
 You can download and update credentials like this
 
-    aws s3 cp s3://contributions-private/DEV/contributions.private.conf /etc/gu
-    aws s3 cp /etc/gu/contributions.private.conf s3://contributions-private/DEV/
+```
+sudo aws s3 cp s3://contributions-private/DEV/contributions.private.conf /etc/gu/ --profile membership
```

Comment:
Yea, I don't like dumping anything accessible only by `sudo`.  I'll probably do a separate PR around that. Once I have something working and can confirm that putting things in `~/.gu` works fine too.